docs/spec/lsm/adr-002.md

```diff
@@ -0,0 +1,362 @@
+  google.protobuf.Timestamp unbonding_time = 9 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  // commission defines the commission parameters.
+  Commission commission = 10 [(gogoproto.nullable) = false];
+  // Number of shares self bonded from the validator
+  string total_validator_bond_shares = 11 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // Total number of shares either tokenized or owned by a liquid staking provider 
+  string total_liquid_shares = 12 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+}
+```
+
+#### Delegation
+The `Delegation` struct's `Exempt` attribute was renamed to `ValidatorBond`
+
+```proto
+// Delegation represents the bond with tokens held by an account. It is
+// owned by one delegator, and is associated with the voting power of one
+// validator.
+message Delegation {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  // delegator_address is the bech32-encoded address of the delegator.
+  string delegator_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // validator_address is the bech32-encoded address of the validator.
+  string validator_address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // shares define the delegation shares received.
+  string shares = 3 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // has this delegation been marked as a validator self bond.
+  bool validator_bond = 4;
+}
+```
+
+### Tracking total liquid stake
+To monitor the progress towards the global liquid staking cap, the module needs to know two things: the total amount of staked tokens and the total amount of *liquid staked* tokens. The total staked tokens can be found by checking the balance of the "Bonded" pool. The total *liquid staked* tokens are stored separately and can be found under the `TotalLiquidStakedTokensKey` prefix (`[]byte{0x65}`). The value is managed by the following keeper functions:
+```go
+func (k Keeper) SetTotalLiquidStakedTokens(ctx sdk.Context, tokens sdk.Dec)
+func (k Keeper) GetTotalLiquidStakedTokens(ctx sdk.Context) sdk.Dec
+```
+
+### Helper functions
+In order to identify whether a liquid stake transaction will exceed either the global liquid staking cap or the validator bond cap, the following functions were added:
+
+```go
+// Check if an account is a owned by a liquid staking provider
+// This is determined by checking if the account is a 32-length module account
+func (k Keeper) AccountIsLiquidStakingProvider(address sdk.AccAddress) bool 
```

Comment:
This was changed in the implementation: DelegatorIsLiquidStaker

Reply:
Fixed in `adr.md`